content/dice_helper.json

```diff
@@ -1,5 +1,5 @@
 {
-    "brawl": {
```

Comment:
Hmm, I'm not sure about this one. I'm pretty sure it broke at some point (I originally had it the way you're now changing it to)

Reply:
I made several tests with original one and translated one.
Also by changing Foundry LANG.
It needs at least a journal delete and a reload to take fully effect.

Reply:
To be clear, are you saying this is a breaking change for anyone already running the module?

Reply:
No, I made it not breaking.
If the skillname in journal starts with "SWFFG." then the skill translation is searched and loaded for building the chat message. But with an old journal (or a handmade one for example) the skillname is loaded as it.
A user with an handmade journal in another langage who has manually tranlated his skills should still work.
But a new user with a foundry configured with another lang can now see the chatmessages (EN by default, FR if set).